docs/hub/enterprise-hub-gating-group-collections.md

```diff
@@ -0,0 +1,166 @@
+# Gating Group Collections
+
+<Tip warning={true}>
+This feature is part of the <a href="https://huggingface.co/enterprise">Enterprise Hub</a>.
+</Tip>
+
+Gating Group Collections allow organizations to grant (or reject) access to all the models and datasets in a collection at once rather than repo per repo. Users will only have to go through **a single access request**.
+
+To enable gating group in a collection:
```

Comment:
```suggestion
To enable group gating for a collection:
```

Reply:
What about "To enable Gating Group in a collection:"? To keep the naming consistent